packages/tizen_package_manager/tizen/src/tizen_package_manager.h

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_PACKAGE_MANAGER_H_
+#define FLUTTER_PLUGIN_PACKAGE_MANAGER_H_
+#include <flutter/event_channel.h>
+#include <flutter/event_sink.h>
+#include <flutter/event_stream_handler_functions.h>
+
+#include "package_manager_utils.h"
+
+class TizenPackageManagerPlugin : public flutter::Plugin {
+ public:
+  using MethodResultPtr =
+      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>>;
+  static void RegisterWithRegistrar(flutter::PluginRegistrar *registrar);
+
+  TizenPackageManagerPlugin();
+
+  virtual ~TizenPackageManagerPlugin();
+
+  flutter::EncodableList packages_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> install_events_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>>
+      uninstall_events_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> update_events_;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Access_Control

You should make data members of a class private. In order to do so, you will want to convert static callback functions (such as `PackageInfoCB`) into static members of the class, for example,

```cpp
// Definition.
class TizenPackageManagerPlugin : public flutter::Plugin {
 private:
  static bool PackageInfoCallback(package_info_h package_info, void *user_data);
...

// Implementation.
bool TizenPackageManagerPlugin::PackageInfoCallback(package_info_h package_info,
                                                    void *user_data) {
```
Another option is to just inline those callbacks where they are used.
```cpp
package_manager_foreach_package_info(
    [](package_info_h package_info, void *user_data) -> bool {
      ...
    }, this);
```

Reply:
OK. I will modify TizenPackageManagerPlugin implementation. it will remove class definition header file and move callback functions to class static functions of the class.

Reply:
Thank you!